doc/source/turkey.md

```diff
@@ -0,0 +1,33 @@
+
+The country’s official language is Turkish. Kurdish is the second most spoken language in the country, but it is not accepted as an official language, there are people who supports this idea, though.
+
+Even though, due to being the most populous city in Turkey, Istanbul is thought to be the capital of the country, the country’s capital is Ankara since the law about the country’s capital was accepted in 1923 by the new parliament of the new country.
+
+If we are to talk about a country, we should talk about their economy. Turkey has a GDP of $794 billion in total which is the 20th biggest one in the world but when you consider the population (GDP per capita) Turkey is 67th in the world. Also, the purchasing power of the Turkish population is the 45th best one in the world.
+
+One of the biggest industries in Turkey is tourism. It has been so beneficial to economy of the country that the tourism has been called “smokeless industry”. Turkey ranked sixth in the world in terms of the number of international tourist arrivals, with over 50 million foreign tourists visiting the country.
+
+Another import point about Turkey is that there are 204 universities in the country. Some thinks this number is unnecessarily high, but the government continues to establish new universities each year. Every city should have at least a university was their main idea regarding this issue, and they have achieved this goal, but they have not stopped yet. There are 840 thousand university students in the country in total and this number is so high that graduating from a university is not important anymore.
+
+The health is thought to be one of the areas where Turkey is good, but the numbers are saying this is not true. In 2018, the total expenditure on health as a share of GDP was the lowest among OECD countries at %6.3 of GDP. Also, Turkey has one of the highest rates of obesity in the world (%29.5).
+
+Due to the fact that the majority of the Turkey’s population is Muslim, there is a common bias about Turkey that is it is an Islamic country, but it is not. Since its foundation, Republic of Turkey is a secular state with no official state religion. The first article of the Turkish constitution says that the country’s regime is republic, and the next article says that Turkey is a secular, social, democratic, nationalist, state of law and it respects human rights. Even though there have always been governments against some of these articles, including the current one, and they have taken some actions to executes their aims, they were not successful, until this one.
```

Comment:
Since the majority of Turkey’s population is Muslim, there is a common bias about Turkey that is it is an Islamic country, but it is not. Since its foundation, the Republic of Turkey is a secular state with no official state religion. The first article of the Turkish constitution says that the country’s regime is a republic, and the next article says that Turkey is a secular, social, democratic, nationalist, state of law and it respects human rights. Even though there have always been governments against some of these articles, including the current one, and they have taken some actions to executes their aims, they were not successful, until this one.